src/analyserwaveform.cpp

```diff
@@ -16,6 +16,7 @@ AnalyserWaveform::AnalyserWaveform(ConfigObject<ConfigValue>* pConfig) :
         m_skipProcessing(false),
         m_waveformData(NULL),
         m_waveformSummaryData(NULL),
+        m_stride(),
```

Comment:
@daschuer Probably a question of how old a compiler we want to allow (-> precise!). All modern version of the common compilers support most of C++11, cf. <http://en.cppreference.com/w/cpp/compiler_support>.